### DISCUSSION TITLE
Projects (beta): fill a 'select one' field from labels

### DISCUSSION THREAD

**Discussion by Totktonada** [community/community#5544] [Projects and Issues]
Say, we have labels that corresponds to teams (historically). It would be nice to add a 'Team' field, which will be updated according to presence of a team label (there is a particular set of such labels).

Ideally it should be a multichoice field (see https://github.com/github/feedback/discussions/5537).

I guess it would also solve https://github.com/github/feedback/discussions/5543 in the intuitive way. The field will be filled from labels automatically, but we can choose its value for notes.

I guess such synchronization of fields with labels would allow to create nice grouping views. Now there are things, which work good with labels, and things, which work good with fields. Sometimes we want both.

Teams are not the only example. Say, we have some status labels like 'in design' or 'ready for review'. We have subsystem labels. A kind of the task ('bug', 'feature', 'code health', 'refactoring'). Now all this information is not available for using in grouping and sorting (simple filtering works, but see also https://github.com/github/feedback/discussions/5533).

**Comment by Jay-o-Way** [comment]
Yeah, we have a huge list of labels available. Now, when creating new projects we use several fields - like "prio", "status", product" and so on. Would be nice if there could be a script to fill in those fields, based on the labels assigned to an issue.
